File: OrientationAPI/Controllers/EmployeesController.cs
using OrientationAPI.Services;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net;
using System.Net.Http;
using System.Web.Http;

namespace OrientationAPI.Controllers
{
	[RoutePrefix("api/employees")]
	public class EmployeesController : ApiController
	{
		[Route(""), HttpGet]
		public HttpResponseMessage ListEmployees()
		{
			var repo = new EmployeeRepository();
			var result = repo.GetAll();
			return Request.CreateListRecordsResponse(result);
		}

		[Route("create"), HttpPost]
		public HttpResponseMessage AddEmployee(EmployeeDto employee)
		{
			var repo = new EmployeeRepository();
			var result = repo.Create(employee);
			return Request.CreateAddRecordResponse(result);
		}

		[Route("employee-details/{employeeId}"), HttpGet]
		public HttpResponseMessage GetEmployeeDetails(int employeeId)
		{
			var repo = new EmployeeRepository();
			var result = repo.GetEmployee(employeeId);
			return Request.CreateResponse(result);
		}

		[Route("employee-details/{employeeId}/training"), HttpGet]
		public HttpResponseMessage GetEmployeeTraining(int employeeId)
		{
			var repo = new EmployeeRepository();
			var result = repo.GetTraining(employeeId);
			return Request.CreateListRecordsResponse(result);
		}

		[Route("{employeeId}"), HttpPut]
		public HttpResponseMessage UpdateEmployee(Employee employee, int employeeId)
		{
			employee.EmployeeId = employeeId;
			var repo = new EmployeeRepository();
			var result = repo.UpdateEmployee(employee);
			return Request.CreateUpdateRecordResponse(result);
		}
	}
}

File: OrientationAPI/App/Controllers/EmployeesController.js
app.controller("EmployeesController", ["$scope", "$http", "$location",
    function ($scope, $http, $location) {
        $scope.header = "Employees";

        $http.get("/api/employees").then(function (result) {
            $scope.employees = result.data;
        }).catch(function (err) {
            console.log(err);
        });

        $http.get("/api/departments").then(function (result) {
            $scope.departments = result.data;
        }).catch(function (err) {
            console.log(err);
        });

        $hhtp.get("/api/employees/employee-details")

        $scope.navigateToCreateEmployee = function () {
            $location.path(`/CreateEmployee`);
        };

        $scope.navigateToEmployees = function () {
            $location.path('/employees');
        };

        $scope.employeeDetails = function () {
            $location.path('/employee-details');
        };

        
    }
]);